package.json

```diff
@@ -81,7 +83,8 @@
     "wc-sass-render": "^1.2.3",
```

Comment:
Update to 1.4

Reply:
`wc-sass-render@1.4.0` doesn't include node-sass in their [package dependency](https://github.com/tristanMatthias/wc-sass-render/blob/1.4.0/package.json), meanwhile this component `npm run sassBuild` script use `node-sass` CLI, so I am not sure we should go with this, or there is something else that I'm missing.
Please advise, @blackfalcon.